src/RefractAPI.cc

```diff
@@ -142,23 +142,25 @@ namespace drafter
     refract::IElement* ParameterValuesToRefract(
         const NodeInfo<snowcrash::Parameter>& parameter, ConversionContext& context)
     {
-        refract::ArrayElement* element = CollectionToRefract<refract::ArrayElement>(
-            MAKE_NODE_INFO(parameter, values), context, LiteralToRefract<std::string>, SerializeKey::Enum);
+        refract::EnumElement* element = new refract::EnumElement;
+
+        refract::ArrayElement* enumerations = CollectionToRefract<refract::ArrayElement>(
```

Comment:
Just personal preference, but I think it would be more readable of the `enumerations` was defined next to where they are set below on line 161. They are not used until that location.

```c++
refract::ArrayElement* enumerations = CollectionToRefract<refract::ArrayElement>(
    MAKE_NODE_INFO(parameter, values), context, LiteralToRefract<std::string>);
element->attributes[SerializeKey::Enumerations] = enumerations;
```

Reply:
Done.